pages/index.js

```diff
@@ -2,36 +2,55 @@ import HeadObject from "../components/head";
 import Nav from "../components/nav";
 import Scene from "../components/scene";
 import Balancer from "react-wrap-balancer";
+import Link from "next/link";
 
 export default function Home() {
   return (
     <div className="dark:text-white dark:bg-black">
-      <HeadObject />
-      <a href="https://hackclub.com"><img src="https://assets.hackclub.com/flag-orpheus-top.svg" className="absolute top-0 left-8 w-1/4 md:w-[14%]" /></a>
+      {/* <HeadObject />
+      <Link href="https://hackclub.com">
+        <img
+          src="https://assets.hackclub.com/flag-orpheus-top.svg"
+          className="absolute top-0 left-8 w-1/4 md:w-[14%]"
+        />
+      </Link> */}
+
+      <div>
+        <a class="banner mx-auto ml-2 z-50" href="https://hackclub.com/">
+          <img
+            src="https://contribute.hackclub.com/images/flag-orpheus-top.png"
+            alt="Hack Club Logo"
+          ></img>
+        </a>
+      </div>
+
```

Comment:
confused on why the switch was made here, the old code works fine!

Reply:
To make it actually work, because it didn't work before.